test/test-helper.ts

```diff
@@ -71,7 +71,21 @@ export class TestDriver {
   public workingDirectory: string;
 
   constructor(public rootDir: string, addToEnv: Record<string, string> = {}) {
-    this.env = Object.assign({ CI: 1 }, process.env, addToEnv);
+    // Node.js prepends all parent node_modules/.bin directories to the PATH env var
+    // which shadows the cdktf CLI we want to test against and set in run-against-dist.sh
+    // While the code is the same, we want the CLI to be installed in a temporary directory
```

Comment:
Isn't this outdated by having a local registry?

Reply:
Nope, we still install in a local directory (but with `--global-style`, which doesn't work from a tar.gz hence the local registry) and need that directory to be used.